vars/main.yml

```diff
@@ -2,6 +2,7 @@
 # vars file for certbot
 certbot_package:
   - certbot
+  - cron
```

Comment:
Cron is called differently for different distributions, you can make something like this:

```yaml
_certbot_packages:
  default:
    - certbot
    - cronie
  Alpine:
    - certbot
    - dcron
  Debian:
    - certbot
    - cron

certbot_packages: "{{ _certbot_packages[ansible_os_family] | default(_certbot_packages['default']) }}"
```

Reply:
thanks @robertdebock  , PR is updated,
Let me know